homeassistant/components/lyric/climate.py

```diff
@@ -223,16 +223,26 @@ def target_temperature(self) -> float | None:
     def target_temperature_high(self) -> float | None:
         """Return the highbound target temperature we try to reach."""
         device = self.device
-        if device.changeableValues.autoChangeoverActive:
-            return device.changeableValues.coolSetpoint
+        if (
+            device.changeableValues.autoChangeoverActive
+            and HVAC_MODES[device.changeableValues.mode] != HVAC_MODE_OFF
+        ):
+            if device.changeableValues.autoChangeoverActive:
+                return device.changeableValues.coolSetpoint
+            return None
```

Comment:
You can merge this logic since the fall-through behavior is already returning None:
```suggestion
            and device.changeableValues.autoChangeoverActive
        ):
            return device.changeableValues.coolSetpoint
```

Reply:
Thank you. I think the changes I just committed make the logic much more concise, but happy to hear your thoughts.

Reply:
Thanks for suggestion @balloob, I think I missed your point initially, but I just made some additional changes to the target_temperature, target_temperature_high, and target_temperature_low logic that I think make them slightly more concise.